### PR TITLE
Improve navigation flow

### PR DESCRIPTION
**Pull request by robbi5**
* using a outlined style for the back button, per material design guidelines
* using a green final save button

Screenshot 2023-08-09 at 12 47 55
Screenshot 2023-08-09 at 12 48 29

### CODE REVIEW

**Comment by raphaelm** [comment]
On my iMin, this crashes with
```
     Caused by: android.content.res.Resources$NotFoundException: String resource ID #0x0
        at android.content.res.Resources.getText(Resources.java:444)
        at android.content.res.Resources.getString(Resources.java:537)
        at androidx.fragment.app.Fragment.getString(Fragment.java:1053)
        at eu.pretix.pretixprint.ui.SettingsFragment.printerSummary(Settings.kt:134)
        at eu.pretix.pretixprint.ui.SettingsFragment.onResume(Settings.kt:112)
        at androidx.fragment.app.Fragment.performResume(Fragment.java:3180)
        at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:606)
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:285)
        at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:113)
        at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1433)
        at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2977)
        at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:2909)
        at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:285)
        at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:334)
        at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:323)
        at androidx.appcompat.app.AppCompatActivity.onPostResume(AppCompatActivity.java:242)
        at android.app.Activity.performResume(Activity.java:8277)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4529)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4574) 
        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52) 
        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2108) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7945) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:603) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
```

**Comment by robbi5** [comment]
sure that this is happening with this PR? the printerSummary modification is over there in the imin pr #56 (i'll have a look)

File: pretixprint/app/src/main/res/layout/fragment_choose_byte_protocol.xml [hunk 1]
```diff
@@ -32,27 +31,11 @@
...
-        android:layout_marginEnd="16dp"
-        android:layout_marginBottom="16dp"
-        android:text="@string/btn_next"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent" />
-
-    <Button
-        android:id="@+id/btnPrev"
-        android:layout_width="wrap_content"
+    <include
```
Comment (raphaelm) [inline pretixprint/app/src/main/res/layout/fragment_choose_byte_protocol.xml:34]: This layout no longer works correctly (shows no choices). previously, as the recyclerview has

app:layout_constraintBottom_toTopOf="@+id/btnNext"

and the include has

app:layout_constraintTop_toBottomOf="@id/list"

Not sure if this is related to the include or due to the constraints themselves

**Review by raphaelm — CHANGES_REQUESTED** [review]

File: pretixprint/app/src/main/res/layout/fragment_choose_byte_protocol.xml (hunk 1 above)
Comment (robbi5) [reply pretixprint/app/src/main/res/layout/fragment_choose_byte_protocol.xml:33]: Yep, somehow accessing btnNext didn't work in all cases. I just gave all the includes their own id, so they can be referenced by `app:layout_constraintBottom_toTopOf` - and it seems to work even on very very small devices

Screenshot 2023-08-10 at 10 15 11

**Review by raphaelm — APPROVED** [review]

**Pull request merged**